solr/core/src/java/org/apache/solr/handler/component/SearchHandler.java

```diff
@@ -726,7 +726,7 @@ private static boolean prepareComponents(
     return true;
   }
 
-  private static String stageInEnglish(int nextStage) {
+  protected String stageToString(int nextStage) {
```

Comment:
How about dropping the `next` prefix?
```suggestion
  protected String stageToString(int stage) {
```